alphas/alpha_functions.py

```diff
@@ -29,3 +29,31 @@ def alpha_001(data: pd.DataFrame, ticker: str) -> pd.DataFrame:
     drop_signal_indices = data["actively_traded"].where(data["actively_traded"] == False).dropna().index
     raw_signal.loc[drop_signal_indices] = 0
     return raw_signal
+
+
+def alpha_042(mult, ite, gt, mean_a, plus, std_a, csrank, div, obv_a, data: pd.DataFrame, ticker: str) -> pd.DataFrame:
```

Comment:
you shouldn't need to pass the functions in as parameters. you're already importing them, so this isn't necessary

Reply:
actually, I don't see where they are being imported

do:

`from alpha_utils import *`

This is generally not accepted, but we will use this since we want the alpha implementations to match their definitions

Reply:
I added it, and adjusted flake8 so that the checker don't disallow a commit